### PR TITLE
Showseed flow + svg icons - #501 #499

### PR DESCRIPTION
**Pull request by jrgarou**
- Change `ModalShowSeed` UI https://github.com/Paratii-Video/paratii-portal/issues/499#issuecomment-384768291
- Use ` <SVGIcon />` instead of `<use/>` tag (include https://github.com/Paratii-Video/paratii-portal/issues/501#issuecomment-384550101)
https://github.com/Paratii-Video/paratii-portal/blob/f5ac48a97fc37680a2b76f122b274d35d9ca5a25/src/scripts/components/foundations/Button.js#L88-L99
- Changes SVGs from Secure Flow

### CODE REVIEW

File: src/scripts/components/Play.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import queryString from 'query-string'
 import { PlaybackLevel } from 'records/PlayerRecords'
 import VideoRecord from 'records/VideoRecords'
 import VideoOverlayContainer from 'containers/VideoOverlayContainer'
-import Button from 'components/foundations/Button'
+import Button, { SVGIcon } from 'components/foundations/Button'
```
Comment (bent0b0x) [inline src/scripts/components/Play.js]: Why is `SVGIcon` part of the `Button.js` file? I think we should separate it. I wouldn't think svg icons need to be coupled to buttons

File: src/scripts/components/Wallet.js [hunk 2]
```diff
@@ -10,9 +9,9 @@ import { ACTIVATE_SECURE_WALLET } from 'constants/ParatiiLibConstants'
 type Props = {
   user: Object,
   userAddress: string,
-  isWalletSecured: boolean,
+  isWalletSecured: string,
```
Comment (bent0b0x) [inline src/scripts/components/Wallet.js]: Hm why is this a `string`? If we are passing in a `string` I think _that_ is the problem. We shouldn't change this flow type

Comment (jrgarou) [reply src/scripts/components/Wallet.js]: this is from @eliawk 
In fact wasn't to be here. Let me change

File: src/scripts/components/Play.js (hunk 1 above)
Comment (jrgarou) [reply src/scripts/components/Play.js]: Totally agree, @bent0b0x

File: src/scripts/components/Wallet.js [hunk 3]
```diff
@@ -12,7 +11,7 @@ type Props = {
   userAddress: string,
   isWalletSecured: boolean,
   closeModal: () => void,
-  openModal: string => void
+  openModal: () => void
```
Comment (eliawk) [inline src/scripts/components/Wallet.js]: openModal need the string of the modal you want to open. why do you change w/ : `() => void`

**Pull request merged**